### PR TITLE
Ability to set team scope using `--team` flag

### PR DESCRIPTION
**Pull request by leo**
This allows the user to select the current scope for deploying. For example:

```
now --team zeit
```

It will only work using the "sh" provider.

### CODE REVIEW

File: src/now.js [hunk 1]
```diff
@@ -435,6 +427,46 @@ const main = async (argv_) => {
     ctx.config.sh = Object.assign(ctx.config.sh, { user })
   }
 
+  if (typeof argv.team === 'string') {
+    const {team} = argv
+
+    if (team.length === 0) {
+      console.log(error(`You defined ${`--team`}, but it's missing a value`))
```
Comment (matheuss) [inline src/now.js]: Let's use `param` :nail_care:

File: src/now.js [hunk 2]
```diff
@@ -393,18 +387,16 @@ const main = async (argv_) => {
...
-    const {argv} = ctx
-    const tokenIndex = argv.indexOf(hasToken) + 1
-    const token = argv[tokenIndex]
+  if (typeof argv.token === 'string') {
+    const {token} = argv
 
-    if (!token) {
-      console.log(error(`You defined ${param(hasToken)}, but it\'s missing a value`))
+    if (token.length === 0) {
+      console.error(error(`You defined ${`--token`}, but it's missing a value`))
```
Comment (matheuss) [inline src/now.js]: Let's use `param` :nail_care:

File: src/now.js [hunk 3]
```diff
@@ -435,6 +427,46 @@ const main = async (argv_) => {
...
+
+    try {
+      const res = await fetch(url, { headers })
+      body = await res.json()
+    } catch (err) {
+      console.error(error('Not able to load teams'))
+      await exit(1)
+    }
+
+    if (body.error) {
```
Comment (matheuss) [inline src/now.js]: Maybe let's check if it's e.g. `HTTP 403` and show `not authorized to access such team` or sth? 🙏

**Review by matheuss — APPROVED** [review]
a-m-a-z-i-n-g

**Comment by leo** [comment]
Just added all of @matheuss's suggestions! ✅

**Pull request merged**